content/md/onboarder/1-set-up-onboarder.md

```diff
@@ -0,0 +1,74 @@
+---
+id: set-up-akeneo-onboarder
+themes: manage-onboarder-instance
+title: "**Set up** your Onboarder instance"
+popular: true
+related: console-activity-history
+---
+
+# Set up your Onboarder instance
+
+The Akeneo Onboarder instance creation is managed on the **projects console** of your [Portal](connect-to-your-portal.html). 
+
+Following the purchase of an Onboarder instance, an email is sent to the **Onboarder administrator** inviting them to complete the setup form to create the instance.
+
+You can set up the instance from the link given in the mail or by accessing the instance configuration page on the **projects console**.
+
+::: info
+The [technical prerequisites](https://docs.akeneo.com/latest/onboarder/prerequisites/index.html#prerequisites) have to be fulfilled before filling the setup form except if you are linking the Onboarder to a Serenity instance.
+:::
+
+![Onboarder list](../img/onboarder_list_waiting_setup.jpg)
+
+1. First, from the menu, go to `Projects console / Projects` and select your project.
+2. Go to the Onboarder instance tab, and open the instance with the status `Waiting for setup`, by clicking on the line of the instance.
+3. Then fill the form with the missing prerequisites in order to create the Onboarder instance.
+
+## Link your Onboarder to an existing PIM Cloud instance
+
+1. Select a `Linked PIM instance` in the list.
+2. Choose a `Domain prefix`, which is going to define of your URL instance (e.g: https://retailer-yourdomainprefix.cloud.akeneo.com). By default, the domain prefix of your linked PIM instance is used in the generated URL. An instant check is done to see if the name is available.
+3. Select an `Installation locale` for your Onboarder.
+4. The `Installation location` is chosen with your linked PIM instance.
+5. Once the information is inputted, click `Request an Instance`.
```

Comment:
Are you sure about "inputted" ? Isn't it derived from "to put" and not "to putt" ?